packages/sit-onyx/src/playwright/screenshots.tsx

```diff
@@ -82,8 +82,9 @@ export const executeMatrixScreenshotTest = async <TColumn extends string, TRow e
       const box = await component.boundingBox();
 
       // accessibility tests
+      // TODO: color-contrast: remove when contrast issues are fixed in https://github.com/SchwarzIT/onyx/issues/410
       const accessibilityScanResults = await makeAxeBuilder()
-        .disableRules(options.disabledAccessibilityRules ?? [])
+        .disableRules(["color-contrast", ...(options.disabledAccessibilityRules ?? [])])
```

Comment:
This should not be needed anymore since the color-contrast is disabled globally in the src/playwright/a11y.ts file